decidim-core/app/controllers/decidim/modals_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ModalsController < Decidim::ApplicationController
+    helper_method :status, :authorize_action_path
+
+    def authorization
```

Comment:
Regarding the naming issue you mentioned, what about changing the controller name to `AuthorizationModalsController` and this action to `show`?

Reply:
:+1: